instance/tests/models/test_instance.py

```diff
@@ -394,13 +395,24 @@ def check_mysql(self, instance):
         try:
             for database in instance.mysql_database_names:
                 self.assertIn(database, databases)
-                mysql_cmd = "mysql -u {user} --password={password} -e 'SHOW TABLES' {db_name}".format(
+                mysql_cmd = (
+                    'mysql'
+                    ' --default-character-set=utf8mb4'
+                    ' --user={user}'
+                    ' --password={password}'
+                    ' {db_name}'
+                ).format(
                     user=instance.mysql_user,
                     password=instance.mysql_pass,
                     db_name=database,
                 )
-                tables = subprocess.call(mysql_cmd, shell=True)
-                self.assertEqual(tables, 0)
+                process = subprocess.Popen(mysql_cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, shell=True)
+                test_sql = ("CREATE TABLE test (value TEXT);"
+                            "INSERT INTO test (value) VALUES ('OHAI 😜');"
+                            "SELECT value FROM test;")
+                output, _ = process.communicate(input=test_sql.encode('utf-8'))
```

Comment:
Might be good to capture stderr as well, and add include it in the error message when the assert in the next line fails (the error message can be given in the third parameter of `assertEqual()`).